models/userDao.js

```diff
@@ -0,0 +1,94 @@
+const appDataSource = require('./dataSource');
+
+const getUserIdByKakaoId = async(kakaoId) => {
+  return await appDataSource.query(`
+    SELECT
+      id AS userId
+    FROM users
+    WHERE kakao_id = ?;`, [ kakaoId ]
+  );
+}
+
+const storeKakaoUserInfo = async(kakaoUserInfo) => {
+  const kakaoId = kakaoUserInfo.id;
+  const nickname = kakaoUserInfo['kakao_account'].profile.nickname;
+  const profileImageUrl = kakaoUserInfo['kakao_account'].profile['profile_image_url'];
+  const email = kakaoUserInfo['kakao_account'].email;
+  const birthday = kakaoUserInfo['kakao_account'].birthday;
+  let gender = kakaoUserInfo['kakao_account'].gender;
+  let genderId = 0;
+
+  if (gender = 'male') {
+    genderId = 1;
+  } else if (gender = 'female') {
+    genderId = 2;
+  } else {
+    genderId = 3;
+  }
+
+  return await appDataSource.query(`
+    INSERT INTO users (
+      kakao_id,
+      nickname,
+      profile_image_url,
+      email,
+      birthday,
+      gender_id
+    ) VALUES (?, ?, ?, ?, ?, ?);`, [ kakaoId, nickname, profileImageUrl, email, birthday, genderId ]
```

Comment:
마이너하지만 배열로 매핑하는 부분 한칸 띄워서 작성해주시죠!

```suggestion
    ) VALUES (?, ?, ?, ?, ?, ?);`,
    [ kakaoId, nickname, profileImageUrl, email, birthday, genderId ]
```

Reply:
넵 해당하는 부분은 모두 개행하고 작성하도록 하곘습니다!